### PR TITLE
chore: Remove PinEditTextField

### PR DESCRIPTION
**Pull request by makingthematrix**
We no longer use PinEditTextField so we can remove the dependency.
The update is motivated by the strive to make us F-droid compliant.

#### APK
[Download build #3908](http://10.10.124.11:8080/job/Pull%20Request%20Builder/3908/artifact/build/artifact/wire-dev-PR3483-3908.apk)
[Download build #3919](http://10.10.124.11:8080/job/Pull%20Request%20Builder/3919/artifact/build/artifact/wire-dev-PR3483-3919.apk)
[Download build #3920](http://10.10.124.11:8080/job/Pull%20Request%20Builder/3920/artifact/build/artifact/wire-dev-PR3483-3920.apk)

### CODE REVIEW

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -66,7 +66,7 @@ object Versions {
     const val COMMON_MARK = "0.11.0"
     const val JNA = "4.4.0@aar"
     const val LIB_PHONE_NUMBER = "7.1.1" // 7.2.x breaks protobuf
-    const val PIN_EDITTEXT = "1.2.1"
+    const val PIN_EDITTEXT = "1.2.3"
```
Comment (mejdoo) [inline buildSrc/src/main/kotlin/Dependencies.kt:69]: Did you check whether the UI is fine or not when we update the lib?

Comment (mejdoo) [reply buildSrc/src/main/kotlin/Dependencies.kt:69]: @makingthematrix

Comment (makingthematrix) [reply buildSrc/src/main/kotlin/Dependencies.kt:69]: @mejdoo : Now that I looked again, I think we don't really use it anymore. It's possible that I can remove the library from dependencies. I will check tomorrow.

Comment (mejdoo) [reply buildSrc/src/main/kotlin/Dependencies.kt:69]: You're right, we can delete it.

**Review by mejdoo — APPROVED** [review]

**Pull request merged**